powertools-core/src/main/java/software/amazon/lambda/powertools/core/internal/EnvironmentVariables.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2023 Amazon.com, Inc. or its affiliates.
+ * Licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package software.amazon.lambda.powertools.core.internal;
+
+public final class EnvironmentVariables {
+    public static class LOGGER {
+        public final static String LOG_EVENT = "POWERTOOLS_LOGGER_LOG_EVENT";
```

Comment:
we already have other files containing environment variables, do not introduce a new one please

Reply:
maybe that's something we should clean up in v2 by the way.